spec/Statements.tex

```diff
@@ -789,15 +789,22 @@ \section{The Use Statement}
 C1, C2, C3 have public module level definitions of the same symbol.
 
 An optional \sntx{limitation-clause} may be provided to limit the symbols made
-available by the use statement.  If an \chpl{except} list is provided, then all
+available by a use statement.  If an \chpl{except} list is provided, then all
 the visible but unlisted symbols in the module or enumerated type will be made
 available without prefix.  If an \chpl{only} list is provided, then just the
 listed visible symbols in the module or enumerated type will be made available
 without prefix.  All visible symbols not provided via these limited use
-statements are still accessible with the module or enumerated type name prefix.
+statements are still accessible by qualifying the access by the module or enumerated type name prefix.
```

Comment:
"by qualifying the access *via* the module . . ." instead?

Reply:
"by...by" _is_ unfortunate.  How about "by qualifying the access using the module or enumerated type name as a prefix." ?

Reply:
I did initial try "using" but was worried about it being confusing with "use" statements, but I'm okay with it if you are!

Reply:
ugh, good point.  How about "by prefixing the access with the name of the module or enumerated type." ?

Reply:
That works for me!